### PR TITLE
WIP branch for issue 87

### PR DESCRIPTION
**Pull request by gregheo**

### CODE REVIEW

File: _drafts/2017-09-14-issue-87.md [hunk 1]
```diff
@@ -23,10 +27,16 @@ sponsor:
 
 ### Swift Unwrapped
 
-> TODO: Latest episode of Swift Unwrapped
+In [episode 28 of Swift Unwrapped](https://spec.fm/podcasts/swift-unwrapped/85798), JP and Jesse discuss the Clang refactoring engine and how you shouldn’t be afraid of C++ code.
```
Comment (jpsim) [inline _drafts/2017-09-14-issue-87.md]: We do talk about the clang side of the refactoring engine, but most of the episode is actually about the Swift one. Suggest to rephrase to:

> JP and Jesse discuss the Clang & Swift refactoring engines and how you shouldn’t be afraid of C++ code.

**Comment by jessesquires** [comment]
looking 🎉 👌

File: _drafts/2017-09-14-issue-87.md [hunk 2]
```diff
@@ -9,19 +9,28 @@ sponsor:
     displaylink: plausible.coop
 ---
 
-> TK details below
+> Foundation improvements, no longer a fable<br>
+> Resilience and ownership, back on the table<br>
+> &nbsp;&nbsp;&nbsp;Swift 4 is soon live<br>
+> &nbsp;&nbsp;&nbsp;Now look to Swift 5<br>
+> The primary goal: get that ABI stable<br>
```
Comment (jpsim) [inline _drafts/2017-09-14-issue-87.md]: dat limerick tho

Comment (jessesquires) [reply _drafts/2017-09-14-issue-87.md]: So good 😍

Comment (gregheo) [reply _drafts/2017-09-14-issue-87.md]: GitHub needs angry reacc for Jesse 😡

File: _drafts/2017-09-14-issue-87.md [hunk 3]
```diff
@@ -32,29 +41,40 @@ In [episode 28 of Swift Unwrapped](https://spec.fm/podcasts/swift-unwrapped/8579
...
+
+Logan Shire posted about [some common issues when working with enumerations](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170904/039598.html): needing to iterate through all options, and getting a count of cases. Again, check out the thread for some discussion on [why you might need these features](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170904/039601.html) and how they might be implemented.
+
 
 ### Finally
 
-> TODO: something funny/fun. tweet, link, etc.
+Q: What color would Chris Lattner paint the [bikeshed](https://en.wiktionary.org/wiki/bikeshedding)? 🎨
+
+A: [Resilience](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170904/039672.html).
```
Comment (jessesquires) [inline _drafts/2017-09-14-issue-87.md]: LOL

**Comment by gregheo** [comment]
@jessesquires I don't need your approval!!! 😈

**Comment by jessesquires** [comment]
lolololololololol 😆

**Comment by gregheo** [comment]
Manually merged in 3033f369.

**Comment by jessesquires** [comment]
Thanks @gregheo !

**Pull request closed without merging**